src/seedu/addressbook/data/person/Phone.java

```diff
@@ -56,4 +57,9 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+    @Override
+    public String getPrintableString() {
+        return isPrivate ? "" : PHONE_PREFIX + value;
```

Comment:
it would be more readable and easier to add new stuff if we use if-else. However, this is still fine. Good job! :D